### PR TITLE
Expand documentation around configuring the PAS tile

### PR DESCRIPTION
**Pull request by XenoPhex**
Expanded documentation that resulted from [**Explore** Use the Minio tile with PAS](https://www.pivotaltracker.com/story/show/170939202)

### CODE REVIEW

**Comment by tehall16** [comment]
@krishnasrinivas Let me know if you want this one merged too.

File: docs-content/operator-guide.html.md.erb [hunk 1]
```diff
@@ -111,6 +111,17 @@ For example if 6 instances are configured with each instance having 200G, total
...
+<li>Configure File Storage in your Application Service Tile<br/>
+  <ul>
+  <li>Under "Configure your Cloud Controller's filesystem" select "External S3-compatible filestore"</li>
+  <li>For "URL endpoint", provide the previously recorded load-balancer IP address in the form of "http://<IP address>"</li>
+  <li>The "Access Key" and "Secret Key" are the previously recoreded AccessKey and SecretKey.</li>
+  <li>The "S3 signature version" should be set to "V2 Signature"</li>
+  <li>The "Region" should be set to "us-east-1".</br><i>Note: Even though Minio has no concept of regions, it maintains this interface to be compatible with AWS S3 and is required to be filled in.</i></li>
+  <li>The Buildpacks, Droplets, Packages, and Resources bucket names should match the ones from the "Buckets" pane in the Minio tile.</li>
+  <li><strong>Warning:</strong> Minio currently does not support versioned backup, if you'd like to setup backups, proceed to the Enable Blobstore Backups Section.</li>
+  </ul>
```
Comment (krishnasrinivas) [inline docs-content/operator-guide.html.md.erb:123]: @XenoPhex thanks for this documentation. Can you align the above steps with the latest documentation. The section `12. Note down the MinIO Internal Blobstore access details` covers these steps.

**Comment by krishnasrinivas** [comment]
> @krishnasrinivas Let me know if you want this one merged too.
@tehall16 I have requested few changes, we can take it in once they are taken care of.

File: docs-content/operator-guide.html.md.erb (hunk 1 above)
Comment (XenoPhex) [reply docs-content/operator-guide.html.md.erb:140]: I'll have time later this week to clean this up.

**Comment by XenoPhex** [comment]
So I'm no longer a VMware/Pivotal employee, so this is unlikely to get done. Also, I think the original requester is no longer employee.

Not sure if this will get done unless y'all handle things yourself at this point. Sorry 😞

**Pull request closed without merging**